### PR TITLE
AddEvidence in JS

### PR DESCRIPTION
**Pull request by gshriki**
Converted to JavaScript
Complying with how we provide lastCompletedTaskEntries with the new server

### CODE REVIEW

**Comment by slavikm** [comment]
@liorkol @gshriki is this done?

**Comment by gshriki** [comment]
done from my side. pending review and merge...

File: Scripts/script-AddEvidence.yml [hunk 1]
```diff
@@ -1,31 +1,41 @@
 commonfields:
-  id: 74eb90a9-cffc-44e1-8fac-6d7e2ddd5b0e
-  version: 12
+  id: 621defb8-bfb6-4f03-898f-699442880b37
```
Comment (slavikm) [inline Scripts/script-AddEvidence.yml]: Same as name and id = -1

File: Scripts/script-AddEvidence.yml [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
+      throw 'Missing argument values for command AddEvidenceJS are : entryIDs';
+  }
+  var entryIDs = args.entryIDs;
+  var desc = (args.description) ? args.description : 'Evidence added by DBot';
+  entryIDs = (Array.isArray(entryIDs)) ? entryIDs : [entryIDs];
+
+  for (var i=0;i<entryIDs.length;i++) {
+      var entryID = entryIDs[i];
+      entries = executeCommand('getEntry', {'id': entryID});
+      entries = (Array.isArray(entries)) ? entries : [entries];
```
Comment (slavikm) [inline Scripts/script-AddEvidence.yml]: I don't like this - I usually do `isResValid(entries)`

File: Scripts/script-AddEvidence.yml [hunk 3]
```diff
@@ -1,31 +1,41 @@
...
+          }
+      }
+  }
+
+  return "Entry ID " + entryIDs + " added to evidence";
+type: javascript
+tags:
+- Utility
+comment: |
+  Adds provided entries to the incident Evidence Board. In playbook. In playbook, can be positioned after a task to add the previous task's entries to Evidence Board automatically (with no need to provide arguments)
```
Comment (slavikm) [inline Scripts/script-AddEvidence.yml]: stutter

File: Scripts/script-AddEvidence.yml [hunk 4]
```diff
@@ -1,31 +1,41 @@
...
+  entryIDs = (Array.isArray(entryIDs)) ? entryIDs : [entryIDs];
+
+  for (var i=0;i<entryIDs.length;i++) {
+      var entryID = entryIDs[i];
+      entries = executeCommand('getEntry', {'id': entryID});
+      entries = (Array.isArray(entries)) ? entries : [entries];
+      for (var j=0;j<entries.length;j++){
+          var ent=entries[j];
+          if (((ent.ContentsFormat == 'table' || ent.ContentsFormat == 'markdown') && ent.Contents) || ent.Type == entryTypes['file']) {
+              executeCommand('markAsEvidence', {'id': entryID, 'description': desc})
```
Comment (slavikm) [inline Scripts/script-AddEvidence.yml]: Check for error, no?

**Pull request merged**